### DISCUSSION TITLE
How to recover Production instance that wasn't backed up?

### DISCUSSION THREAD

**Discussion by lfg-seth** [inventree/InvenTree#5780] [General]
Writing this as a discussion and not an issue since the problem is because of me and not the code base. 

I had to do a rather hasty migration of my home labs vms and in my haste I just cold copied all the files from the vm running InvenTree instead of exporting or backing up the database. I was running the production docker instance from [this guide](https://docs.inventree.org/en/latest/start/docker_prod/). I installed a new ubuntu server vm, copied over the files as they were, tried to run docker compose up and kept getting db errors. So I ran the `docker compose run inventree-server invoke update` and now it works...but theres no parts, stock, or any data.  Is there a way I can recover the data?

**Comment by lfg-seth** [comment]
Welp. I just tried it all again and now theres data. So I guess Im good. Time to implement regular backups.....

**Reply by matmair** [reply]
It can take a recovering postgresql database container a while to consolidate the WAL.
Regular backups are probably a good idea. If you already have a solution for psql in place you would just need to archive media files, plugin.txt, secret.txt and config.yaml file
